content/engine/release-notes/27.2.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Docker Engine 27.2 release notes
+description: Learn about the new features, bug fixes, and breaking changes for Docker Engine
+keywords: docker, docker engine, ce, whats new, release notes
+toc_min: 1
+toc_max: 2
+tags:
+  - Release notes
+aliases:
+- /engine/release-notes/
+- /engine/release-notes/latest/
+- /release-notes/docker-ce/
+- /release-notes/docker-engine/
+---
+
+This page describes the latest changes, additions, known issues, and fixes for Docker Engine version 27.2.
+
+For more information about:
+
+- Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
+- Changes to the Engine API, see [Engine API version history](/reference/api/engine/version-history.md).
+
+## 27.2.0
+
+{{< release-date date="2024-08-27" >}}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 27.2.0 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A27.2.0)
+- [moby/moby, 27.2.0 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A27.2.0)
+- Deprecated and removed features, see [Deprecated Features](https://github.com/docker/cli/blob/v27.2.0/docs/deprecated.md).
+- Changes to the Engine API, see [API version history](https://github.com/moby/moby/blob/v27.2.0/docs/api/version-history.md).
+
+### New
+
+The new features in this release are:
+
+- [Device code login](#device-code-login)
+- [Multi-platform support for `docker image ls`](#multi-platform-support-for-docker-image-ls)
+
+#### Device code login
+
+This release adds support for using device code login when authenticating to
+Docker Hub.
+
+You can still use the old method of logging in with a username and password or
+access token, but device code login is more secure and doesn't require you to
+enter your password in the CLI.
+
+![device-code-login](https://i.imgur.com/GRGjSjV.gif "Device code login")
+
+To use the old method, use `docker login -u <username>`.
+
+#### Multi-platform support for `docker image ls`
+
+{{< experimental >}}
+This is experimental and may change at any time without any backward compatibility.
+{{< /experimental >}}
+
+With the containerd image store enabled, the `docker image ls` command (or
+`docker images` shorthand) now supports a `--tree` flag that now shows
+if an image is a multi-platform image.
+
+<script src="https://asciinema.org/a/PyE57rUPdNh7LvOMhULtE0Ek2.js" id="asciicast-PyE57rUPdNh7LvOMhULtE0Ek2" async="true"></script>
```

Comment:
I guess somewhat similar question here; looks like it can be self-hosted, but not 100% sure what amount of server-side is needed to show these; https://docs.asciinema.org/manual/server/self-hosting/